packages/cli/src/message-format.js

```diff
@@ -10,11 +10,11 @@ export const formatMessage = (strings, edgeNames) => {
     index < Math.min(strings.length, edgeNames.length);
     index += 1
   ) {
-    message += strings[0].replace(/@/g, '\\@');
+    message += strings[index].replace(/@/g, '\\@');
     message += `@${edgeNames[index]}`;
   }
   if (strings.length > edgeNames.length) {
-    message += strings[index];
+    message += strings[index].replace(/@/g, '\\@');
```

Comment:
Obvious error in hindsight. Tests for the win. This will become more complicated when we support dereferencing dotted paths through name hubs.